extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/bridge.xml

```diff
@@ -32,7 +32,7 @@
 				<label>Polling Interval</label>
 				<description>Seconds between fetching values from the Bridge.</description>
 				<required>true</required>
-				<default>10</default>
+				<default>1</default>
```

Comment:
Note that there are two more occurrences of the ten second default value, one in HueBridgeConfig and one in HueBridgeHandler. When going with this approach, we might want to change them as well to stay consistent.

Reply:
That is not very consistent. However, I think this is the route to go. 
Once upon a time we set this value at runtime and suffered from the fact that it was not applied instantaneously. Instead a restart of the framework was required to be applied. 

I'm going to change values in HueBridgeConfig and HueBridgeHandler, too.